handpose/demo/index.js

```diff
@@ -15,8 +15,17 @@
  * =============================================================================
  */
 
+import * as tfwebgpu from '@tensorflow/tfjs-backend-webgpu';
+import * as tf from '@tensorflow/tfjs-core';
 import * as handpose from '@tensorflow-models/handpose';
-
+import * as tfjsWasm from '@tensorflow/tfjs-backend-wasm';
+// TODO(annxingyuan): read version from tfjsWasm directly once
+// https://github.com/tensorflow/tfjs/pull/2819 is merged.
+import {version} from '@tensorflow/tfjs-backend-wasm/dist/version';
```

Comment:
#2819 should have been merged, so you should be able to read the version directly from tfjsWasm now instead of fetching it from dist.

Reply:
Thanks Ann, I followed up it here: https://github.com/tensorflow/tfjs-models/pull/442